QgisModelBaker/libqgsprojectgen/dataobjects/form.py

```diff
@@ -110,8 +110,9 @@ def create(self, parent, layer):
 
 class FormRelationWidget(object):
 
-    def __init__(self, relation):
+    def __init__(self, relation, nmRelation=None):
         self.relation = relation
+        self.nmRelation = nmRelation
```

Comment:
Can you avoid camelcase in variable names https://www.python.org/dev/peps/pep-0008/#function-and-variable-names ?

Reply:
Yes, it was the habit from c++